recipes/paleomix/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Is this build script needed? Can’t you use the normal way to install Python packages instead? That is, use this in the `build` section in the recipe:
```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```

Reply:
pip is kind of broken for paleomix... I never successfully installed it using pip.